### ISSUE TITLE
New Expensify Supportal is showing account information for the responder when Supportalled into an account

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#50673]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @VictoriaExpensify 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1728428277760989

## Action Performed:

1. Open Supportal
2. Enter the customer/test email address and sign in
3. Select the New Supportal tab
4. Click around the account a bit. In this instance, I clicked into Search > Chats.

## Expected Result:
I should only see customer data when supportalled into the customer account

## Actual Result:
- At first I had the grey spinner 
- Then when it loaded, it loaded the chats from my own account. 
- Now, it is showing the details of my own in Expenses and Inbox, but it still looks like I'm supportalled into the customer account

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos
![2024-10-09_08-33-18 (1)](https://github.com/user-attachments/assets/09c57a17-8f1a-4bd8-be78-9fdb59e95aee)

<details>
  <summary>Add any screenshot/video evidence</summary>

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by MelvinBot** [comment]
This has been labelled "Needs Reproduction". Follow the steps here: https://stackoverflowteams.com/c/expensify/questions/16989

**Comment by bfitzexpensify** [comment]
I'm having trouble reproducing myself. 

Is this something you're getting with all accounts you supportal into @VictoriaExpensify?

**Comment by bfitzexpensify** [comment]
Bump on the q above @VictoriaExpensify

**Comment by VictoriaExpensify** [comment]
Oh yeah, this is a bit tricky to reproduce because it doesn't happen consistently. I'm going to post this in #concierge_management and ask that examples get added here

**Comment by bfitzexpensify** [comment]
Switching this to a weekly while we wait on more reports - I think this is a bug worth fixing, but it would be nice to have it occur more than once so we can actually determine it is ongoing

**Comment by bfitzexpensify** [comment]
Still no further reports. @VictoriaExpensify not sure if you recall, but did you resolve this with a clean sign out / refresh or something similar? 

If so, I think we could maybe move this down to Polish @maddylewis - we haven't had any other reports, and if the fix for it happening was simple, I don't think it's that important to look into further against everything else we're working on.
